tests/auth_tests/test_context_processors.py

```diff
@@ -33,6 +33,10 @@ def __eq__(self, other):
             self.eq_calls += 1
             return False
 
+    def test_repr(self):
+        perms = PermWrapper(MockUser())
+        self.assertEqual(repr(perms), f'{perms.__class__.__qualname__} user={perms.user!r}')
```

Comment:
Please compare to the expected value, e.g. 'PermWrapper(...)'.

Reply:
I have updated the values as you asked for, but I'm curious to know if checking it be to `PermWrapper` is better than `perms.__class__.__name__`? Wouldn't we have to change the tests, if by any chance we change the class name?

I'm just asking this to know which is kind of a better design for a test. I tend to use the non-constant form till now in my tests.

Reply:
> Wouldn't we have to change the tests, if by any chance we change the class name?

Yes and it's expected, we will be aware which tests are affected by our change to the base class. IMO it's also more readable.